#/***********************************************************************
# * Licensed Materials - Property of IBM 
# *
# * IBM SPSS Products: Statistics Common
# *
# * (C) Copyright IBM Corp. 1989, 2011
# *
# * US Government Users Restricted Rights - Use, duplication or disclosure
# * restricted by GSA ADP Schedule Contract with IBM Corp. 
# ************************************************************************/

# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR ORGANIZATION
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"POT-Creation-Date: 2010-01-08 09:15+Mountain Standard Time\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: ENCODING\n"
"Generated-By: pygettext.py 1.5\n"


#: SPSSINC_COMPARE_DATASETS.py:113
msgid "---"
msgstr "---"

#: SPSSINC_COMPARE_DATASETS.py:131
msgid "Warnings "
msgstr "警告 "

#: SPSSINC_COMPARE_DATASETS.py:147 SPSSINC_COMPARE_DATASETS.py:205
msgid ""
"The following required parameters were not supplied:\n"
msgstr ""
"次の必須パラメータが指定されませんでした。\n"

#: SPSSINC_COMPARE_DATASETS.py:162
msgid "IBM SPSS Statistics error, possibly invalid dataset name (case matters)"
msgstr "IBM SPSS Statistics エラーです。データセット名が無効であると考えられます (大文字・小文字の区別)"

#: SPSSINC_COMPARE_DATASETS.py:219
msgid "This module requires at least SPSS Statistics 17"
msgstr "このモジュールには SPSS Statistics 17 以上が必要です"

#: SPSSINC_COMPARE_DATASETS.py:391
msgid "format"
msgstr "形式"

#: SPSSINC_COMPARE_DATASETS.py:391
msgid "type"
msgstr "タイプ"

#: SPSSINC_COMPARE_DATASETS.py:391
msgid "variable label"
msgstr "変数ラベル"

#: SPSSINC_COMPARE_DATASETS.py:392
msgid "missing values"
msgstr "欠損値"

#: SPSSINC_COMPARE_DATASETS.py:392
msgid "value labels"
msgstr "値ラベル"

#: SPSSINC_COMPARE_DATASETS.py:393
msgid "attributes"
msgstr "属性"

#: SPSSINC_COMPARE_DATASETS.py:393
msgid "measurement level"
msgstr "尺度"

#: SPSSINC_COMPARE_DATASETS.py:394
msgid "alignment"
msgstr "位置合わせ"

#: SPSSINC_COMPARE_DATASETS.py:394
msgid "column width"
msgstr "列幅"

#: SPSSINC_COMPARE_DATASETS.py:394
msgid "index"
msgstr "インデックス"

#: SPSSINC_COMPARE_DATASETS.py:416
msgid "Active Dataset"
msgstr "アクティブ データセット"

#: SPSSINC_COMPARE_DATASETS.py:425
msgid "Comparison of Datasets %s and %s."
msgstr "データセット %s と %s の比較。"

#: SPSSINC_COMPARE_DATASETS.py:427
msgid ""
"\n"
"Selected Variables"
msgstr ""
"\n"
"選択された変数"

#: SPSSINC_COMPARE_DATASETS.py:433
msgid "A dataset cannot be compared to itself"
msgstr "データセットを自らと比較することはできません"

#: SPSSINC_COMPARE_DATASETS.py:443
msgid "Warning: the following requested variables to be compared are not present in both datasets and will be omitted."
msgstr "警告: 比較する次の要求された変数がデータセットにありません。除外されます。"

#: SPSSINC_COMPARE_DATASETS.py:449
msgid "Warning: the following variables are present in only one dataset and will be ignored."
msgstr "警告: 次の変数が 1 つのデータセットにしかありません。無視されます。"

#: SPSSINC_COMPARE_DATASETS.py:453
msgid "There are no variables to compare"
msgstr "比較する変数がありません。"

#: SPSSINC_COMPARE_DATASETS.py:465
msgid "Count of Value Differences for %d Variables"
msgstr "%d の変数の値の相違数"

#: SPSSINC_COMPARE_DATASETS.py:470
msgid "diffcount variable already exists or invalid name: %s"
msgstr "diffcount 変数 %s が既に存在するか、名前が無効です。"

#: SPSSINC_COMPARE_DATASETS.py:481
msgid "Difference for Variable %s"
msgstr "変数 %s の差"

#: SPSSINC_COMPARE_DATASETS.py:490
msgid "The ID variable specified was not found: %s"
msgstr "指定した ID 変数 %s が見つかりませんでした。"

#: SPSSINC_COMPARE_DATASETS.py:495
msgid "Comparison of Cases"
msgstr "ケースの比較"

#: SPSSINC_COMPARE_DATASETS.py:495
msgid "Count"
msgstr "カウント"

#: SPSSINC_COMPARE_DATASETS.py:497
msgid "No ID variable was specified.  Cases cannot be compared."
msgstr "ID 変数が指定されませんでした。  ケースを比較できません。"

#: SPSSINC_COMPARE_DATASETS.py:515 SPSSINC_COMPARE_DATASETS.py:524
msgid "Duplicate or out of order ID value found in %s.  Processing Stopped"
msgstr "%s 内の ID 値が重複しているか、順序に反しています。処理が停止しました"

#: SPSSINC_COMPARE_DATASETS.py:561
msgid "Cases in Dataset %s"
msgstr "データセット %s のケース"

#: SPSSINC_COMPARE_DATASETS.py:562
msgid "Cases not Matched with Dataset %s"
msgstr "ケースがデータセット %s と一致しません"

#: SPSSINC_COMPARE_DATASETS.py:564
msgid "Dataset %s count of cases not in dataset %s at least"
msgstr "ケースのデータセット %s のカウントが少なくともデータセット %s にありません"

#: SPSSINC_COMPARE_DATASETS.py:565
msgid "Cases with Value Differences"
msgstr "値の差のあるケース"

#: SPSSINC_COMPARE_DATASETS.py:613
msgid "Comparisons: "
msgstr "比較: "

#: SPSSINC_COMPARE_DATASETS.py:614
msgid "Comparison of Variable Dictionaries"
msgstr "変数ディクショナリの比較"

#: SPSSINC_COMPARE_DATASETS.py:615
msgid "Properties That Differ"
msgstr "異なるプロパティ"

#: SPSSINC_COMPARE_DATASETS.py:629
msgid "Variables with Differences"
msgstr "差のある変数"

msgid "Compare datasets"
msgstr "データセットの比較"
